scripts/pum

```diff
@@ -506,6 +524,11 @@ if __name__ == "__main__":
                  'rules'])
     parser_test_and_upgrade.add_argument(
         '-N', '--exclude-schema', help='Schema to be ignored.', action='append')
+    parser_test_and_upgrade.add_argument(
+        '-u', '--max-version', help='upper bound limit version')
+    parser_test_and_upgrade.add_argument(
+        '-v', '--var', nargs=2, help='Assign variable for running SQL deltas.',
+        action='append', required=False)
```

Comment:
Two pretty minor comments here:

1. `required=False` is not necessary.

2. You could use `default=[]` here. In this way the `args.var` variable will always reference a list, and you won't need to test whether `args.var` is not `None` further down in the code.